README.md

```diff
@@ -77,106 +68,46 @@ A penalty for using forceDeallocate can be set per adapter, of up to 2%.
-
-This gate is critical because it can prevent people from receiving their assets upon withdrawals.
-
-**Send Assets Gate** (`sendAssetsGate`): Controls permissions related to sending assets.
-
-Upon `deposit`/`mint`, `canSendAssets` must return true for  `msg.sender` must pass the `canSendAssets` check.
+VaultV2 depositors are charged settable performance fees, a cut on interest (capped at 50%), and management fees (capped at 5%), a cut on principal, that goes to recipient set by the curator.
 
 ### Roles
 
-#### Owner
-
-The owner's role is to set the curator and sentinels.
+- **Owner**: The owner's role is to set the curator and sentinels.
+It can also set the name and symbol of the vault.
 Only one address can have this role.
 
-It can:
-
-- Set the owner.
-- Set the curator.
-- Set sentinels.
-- Set the name.
-- Set the symbol.
-
-#### Curator
-
-The curator's role is to curate the vault, meaning setting risk limits, gates, allocators, fees.
+- **Curator**: The curator's role is to curate the vault, meaning setting [adapters](#adapters), [risk limits](#caps), [gates](#gates), [allocators](#allocator), [fees](#fees).
 Only one address can have this role.
 
-Curator actions are timelockable, except decreaseAbsoluteCap and decreaseRelativeCap.
-Once the timelock has passed, the action can be executed by anyone.
-
-It can:
-
-<a id="curator-timelocks"></a>
-
-- [Timelockable] Increase absolute caps.
-- Decrease absolute caps.
-- [Timelockable] Increase relative caps.
-- Decrease relative caps.
-- [Timelockable] Set the adapter registry.
-- [Timelockable] Set adapters.
-- [Timelockable] Set allocators.
-- [Timelockable] Increase timelocks.
-- [Timelocked by the timelock being decreased] Decrease timelocks.
-- [Timelockable] Set the `performanceFee`.
-  The performance fee is capped at 50% of generated interest.
-- [Timelockable] Set the `managementFee`.
-  The management fee is capped at 5% of assets under management annually.
-- [Timelockable] Set the `performanceFeeRecipient`.
-- [Timelockable] Set the `managementFeeRecipient`.
-  increaseTimelock should be used carefully, because decreaseTimelock is timelocked with the timelock itself. In particular it is possible to make an action irreversible (which is a feature in itself). A timelock of `type(uint256).max` is a recommended convention for making an action irreversible.
-
-#### Allocator
-
-The allocators' role is to handle the allocation of the liquidity (inside the caps set by the curator).
+- **Allocator(s)**: The allocators' role is to handle the vault's allocation in and out of underlying protocols (with the enabled adapters, and inside the caps set by the curator).
 They are notably responsible for the vault's liquidity.
-Multiple addresses can have this role.
-
-It can:
-
-- Allocate funds from the “idle market” to enabled markets.
-- Deallocate funds from enabled markets to the “idle market”.
-- Set the `liquidityAdapter` and the `liquidityData`.
-- Set the `maxRate`.
```

Comment:
Same here, I would keep it or at least incorporate the last 2 items (liquidity adapter, max rate) in the description for completeness.

Reply:
right, mb